### PR TITLE
Progress

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: demo/pages/progress/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Controller.$inject = ['$http', 'qgrid', '$timeout'];
+export default function Controller($http, qgrid, $timeout) {
+	const ctrl = this;
+	ctrl.gridModel = qgrid.model();
+	ctrl.gridModel.progress({isBusy: true});
+	ctrl.gridModel.data({
```
Comment (klumba12) [inline demo/pages/progress/index.js]: dont need columns as you use generation="deep"

File: src/ng/plugins/progress/progress.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<md-progress-linear md-mode="indeterminate" ng-if="$progress.busy"></md-progress-linear>
```
Comment (klumba12) [inline src/ng/plugins/progress/progress.html]: can't use md- in default template use it in themes/ng

File: src/ng/plugins/progress/progress.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		super(...arguments);
+	}
+
+	get busy() {
+		return this.model.progress().isBusy;
+	}
+
+	onInitCore() {
+		this.model.progressChanged.on(() => {
+			if (this.model.progress().isBusy) {
```
Comment (klumba12) [inline src/ng/plugins/progress/progress.js]: Need to check changes.isbusy

File: src/ng/plugins/progress/progress.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	constructor() {
+		super(...arguments);
+	}
+
+	get busy() {
+		return this.model.progress().isBusy;
+	}
+
+	onInitCore() {
+		this.model.progressChanged.on(() => {
```
Comment (klumba12) [inline src/ng/plugins/progress/progress.js]: Use .watch instead of .on,  ut cant be already busy

File: demo/pages/column-pin/index.js [hunk 5]
```diff
@@ -8,7 +8,6 @@ export default function Controller($http) {
 		const props = prop.split('.');
 		let result = item;
 		for (let value of props) {
-			// alert(item.pin);
```
Comment (titovmx) [inline demo/pages/column-pin/index.js:11]: debug code? please remove before merge if it's not neccessary

Comment (etokareva) [reply demo/pages/column-pin/index.js:11]: This is exactly what I've done here! :-) This is 'before' code

**Review by klumba12 — APPROVED** [review]

**Pull request merged**